examples-next/basic/schema.ts

```diff
@@ -229,7 +229,7 @@ export const extendGraphqlSchema = graphQLSchemaExtension({
       createRandomPosts(root, args, context) {
         // TODO: add a way to verify access control here, e.g
         // await context.verifyAccessControl(userIsAdmin);
-        const lists: KeystoneListsAPI<KeystoneListsTypeInfo> = context.lists;
+        const lists = context.lists as KeystoneListsAPI<KeystoneListsTypeInfo>;
```

Comment:
I'd like to get @mitchellhamilton to check out what's going on here, and why this is necessary

Reply:
In the `ecommerce example` we have:

```
type ListsAPI = KeystoneListsAPI<any /* KeystoneListsTypeInfo */>;
...
const lists: ListsAPI = context.lists;
```

which we would also probably want to get to the bottom of.

Reply:
That one in the `ecommerce` example is because I didn't want to commit the types, which we can fix by having a type generation process that gets run by CI before linting. Mitchell has thoughts on this, for now we should probably just commit the types so this isn't commented out.

Reply:
> In the ecommerce example we have:

That's probably because the generated types file for the ecommerce example is gitignored, it shouldn't be gitignored

> needing to use `as`

https://github.com/keystonejs/keystone/pull/4424/files#r532976085 will make it unnecessary to use `as`(which is a good thing because `as` is nuanced and you should avoid it where possible(except for `as const` which is totally different and v good))